src/_glass.scss

```diff
@@ -85,14 +95,15 @@
 }
 
 // D/Glass/Materials/Raised
-@mixin glass-materials-raised {
+@mixin glass-materials-raised($startColor: rgba(167, 163, 163, 0.05), $endColor: rgba(167, 163, 163, 0.05)) {
```

Comment:
I'm a little confused as to why we had to add parameters here. Nothing is using these.